app/src/main/java/com/example/puzzlepic/ui/main/MainFragment.kt

```diff
@@ -133,4 +129,8 @@ class MainFragment : Fragment() {
             currentPhotoPath = absolutePath
         }
     }
+
+    companion object {
+        fun newInstance() = MainFragment()
+    }
```

Comment:
Here, I changed the placement of the companion object and refactored the usage of the button name changed mentioned in my rationales. I referenced the class layout rule here - https://kotlinlang.org/docs/coding-conventions.html#class-layout